easybuild/easyconfigs/b/BOINC/BOINC-7.16.5-foss-2019b-client.eb

```
@@ -0,0 +1,48 @@
+easyblock = 'MakeCp'
+
+name = 'BOINC'
+version = '7.16.5'
+versionsuffix = "-client"
+
+homepage = 'https://boinc.berkeley.edu'
+description = """BOINC is a program that lets you donate your idle computer time to science projects
+ like SETI@home, Climateprediction.net, Rosetta@home, World Community Grid, and many others."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/BOINC',
+        'repo_name': 'boinc',
+        'tag': 'client_release/%(version_major_minor)s/%(version)s',
+    },
+}]
+checksums = ['d8c808682d0549b7e1045f1b4633aeb8287619014effc4c16c7b4145b089f988']
+
+# Depencencies given at https://boinc.berkeley.edu/trac/wiki/SoftwarePrereqsUnix
+# gcc-c++, autoconf, openssl-devel, automake, libtool, libcurl-devel
+
+builddependencies = [
+    ('Autotools', '20180311'),
+]
+
+dependencies = [
+    ('cURL', '7.66.0'),
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libcurl-devel'),
```

Typo? cURL is a anormal dependency. libopenssl-devel is the alternative ssl package name on some distros 
```suggestion
    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
```

---

There is a dependence on `libcurl-devel`.  Without it, `configure` throws the following error:

    ERROR: could not find (recent enough) development-libs for libcurl

Isn't it right to include this as an OS dependency?

Regarding the name of SSL devel package, doesn't EB do some magic to find the correct package name for the platform?

---

1. It should (we should make it) find the libcurl headers from the cURL easyconfig dependency.

2. The only "magic" is that you specify a tuple of alternative names, openssl-devel, libssl-dev, libopenssl-devel are the alterantive names. libcurl-devel does not belong in the same tuple. If we also had a libcurl os dependency (which I don't think we should) we should specify a seperate tuple with it's alternative names, like
```
osdependencies = [
    ('libcurl-devel', 'libcurl-dev'),
    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
]
```
Though again; it's already a normal dependency, it shouldn't also be a osdependency.